Web/ImageTracker/src/WebARImageTracker.tsx

```diff
@@ -73,70 +74,81 @@ export function WebARImageTracker(props: WebARImageTrackerContainerProps): React
     }, []);
 
     useEffect(() => {
-        if(engineContext.scene){
+        if(engineContext.scene && !observableSet){
```

Comment:
What is the main change here? Why we need this observableSet?

Reply:
Oh this is to make sure this thing only happens once. It wasn't getting called or was getting called too often in a previous iteration.